wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/ConversationAvatarView.swift

```diff
@@ -166,7 +166,7 @@ final class ConversationAvatarView: UIView {
                 return
             }
 
-            accessibilityLabel = "Avatar for \(conversation.displayName)"
+            accessibilityLabel = "Avatar for \(conversation.displayNameWithFallback)"
```

Comment:
Which warning was this?

Reply:
String interpolation produces a debug description for an optional value; did you mean to make this explicit?

Reply:
What's the fallback?

Reply:
@johnxnguyen 

```swift
extension ConversationLike {

    var displayNameWithFallback: String {
        return displayName ?? L10n.Localizable.Profile.Details.Title.unavailable
    }

}
```

The fallback is `Name not Available` as recently introduced by Katerina! I guess we can use that. and that's why I'm using it.